### ISSUE TITLE
Release 0.5

### ISSUE THREAD

**Issue by benjaoming** [django-wiki/django-wiki#973]
Nothing big seems to be challenging this release, so let's get it out for Django 2.2+3 support not least

**Comment by Joaxin** [comment]
hahaha, I need this release soon!

```
ERROR: wiki 0.4.4 has requirement Django<2.2,>=1.11, but you'll have django 2.2.
6 which is incompatible.
ERROR: wiki 0.4.4 has requirement django-mptt<0.10,>=0.9, but you'll have django
-mptt 0.10.0 which is incompatible.
ERROR: wiki 0.4.4 has requirement Markdown<2.7,>=2.6, but you'll have markdown 3
.1.1 which is incompatible.
```

Will wiki support python 3.8?

**Comment by Joaxin** [comment]
Also I suggest support `django-markdown-editor` [django-markdown-editor](https://github.com/agusmakmun/django-markdown-editor)

**Comment by benjaoming** [comment]
Hi @Joaxin - thanks for pinging this :) I would suggest to look at existing issues regarding editors. Also closed ones.

As for Python 3.8, it's not in the current Travis matrix and should be dealt with in a separate issue.

**Comment by jschrewe** [comment]
Is there any update in this? Support for Django 2.2 would be nice!

**Comment by benjaoming** [comment]
Thanks @jschrewe for the reminder, I have released it.

Would appreciate quick help and feedback if there are any issues with the release!

**Comment by benjaoming** [comment]
Released in #991
